app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.controller.js

```diff
@@ -360,13 +365,20 @@ module.exports = angular.module('spinnaker.loadBalancer.aws.create.controller',
     this.submit = function () {
       var descriptor = isNew ? 'Create' : 'Update';
 
-      $scope.taskMonitor.submit(
-        function() {
-          return formatListeners().then(function () {
-            return loadBalancerWriter.upsertLoadBalancer($scope.loadBalancer, application, descriptor);
-          });
-        }
-      );
+      if ($scope.forPipelineConfig == true) {
```

Comment:
This should be `===`. Or you can omit the right hand side altogether and just go with:
```javascript
if (forPipelineConfig) {
```